mobile/src/actions/mediaBackup.js

```diff
@@ -33,8 +45,11 @@ export const mediaBackup = (dir) => async (dispatch, getState) => {
     let photos = await getFilteredPhotos()
     const alreadyUploaded = getState().mobile.mediaBackup.uploaded
     const dirID = await getDirID(dir)
+    const totalUpload = photos.length - alreadyUploaded.length
```

Comment:
It's not good here.
I think you can filter photos like that:

    photos = photos.filter(photo => !alreadyUploaded.includes(photo.id))
    const totalUpload = photos.length

and remove `if (!alreadyUploaded.includes(photo.id)) {` line 51